src/lib/js-split-provider.ts

```diff
@@ -130,6 +132,44 @@ export class OpenFeatureSplitProvider implements Provider {
     }
   }
 
+  async track(
+    trackingEventName: string,
+    context: EvaluationContext,
+    details: TrackingEventDetails
+  ): Promise<void> {
+
+    // targetingKey is always required
+    const { targetingKey } = context;
+    if (targetingKey == null || targetingKey === "")
+      throw new TargetingKeyMissingError();
+
+    // eventName is always required
+    if (trackingEventName == null || trackingEventName === "")
+      throw new ParseError("Missing eventName, required to track");
+
+    // trafficType is always required
+    const ttVal = context["trafficType"];
+    const trafficType =
+      ttVal != null && typeof ttVal === "string" && ttVal.trim() !== ""
+        ? ttVal
+        : null;
+    if (trafficType == null || trafficType === "")
+      throw new ParseError("Missing trafficType variable, required to track");
+
+    let value = 0;
```

Comment:
`value` should not be `0` by default. We should support the default case where the user doesn't want to send a `value` with the event. For example when calling track without value: `client.track(targettingKey, trafficType, trackingEventName);`